eng/pipelines/templates/jobs/archetype-sdk-mgmt.yml

```diff
@@ -5,9 +5,11 @@ jobs:
     steps:
       #- script: "echo $(system.pullrequest.pullrequestnumber), https://github.com/$(build.repository.id), https://github.com/$(build.repository.ID)"
       - template: /eng/pipelines/templates/steps/install-dotnet.yml
-      - script: |
-          pwsh eng/scripts/Track1CodeCheck.ps1
+      - pwsh: |
+          eng/scripts/Track1CodeCheck.ps1
```

Comment:
Why not move the entire Track1CodeCheck to a later step instead of wrapping it like this? Does the order in which it is called have impact on the other steps?

Reply:
Sure, the only step which could wrap this part is "Build & Package", so i put it behind current scripts. No imoact found.